apps/design/shared/src/layout.ts

```diff
@@ -485,14 +481,17 @@ function Contest({
 }): [Rectangle, GridPosition[]] {
   assert(contest.type === 'candidate');
 
-  const titleLines = textWrap(
-    contest.title,
-    gridWidth(CONTEST_COLUMN_WIDTH - 0.5)
-  );
+  // Temp hack until we can change the timing mark grid dimensions: expand the
+  // last column to fill the page
```

Comment:
Took me a second to register that "last column" here means the last overall ballot column, not the last grid column. A quick addition to the comment could help!

Reply:
Thanks, will update!